cogs/utility.py

```diff
@@ -53,6 +53,52 @@ async def botcollector(self, ctx, day: int, month: int):
             output = "No one found!"
         await message.edit_original_response(content=output)
 
+    @bridge.bridge_command(brief="Announce something!")
+    @option("channel", discord.TextChannel, description="The channel to announce in")
+    @option("message", str, description="The message to announce")
+    @option("embed", bool, description="Whether to make it an embed")
+    @bridge.has_permissions(manage_guild=True)
+    async def announce(self, ctx, channel: discord.TextChannel, message: str, embed):
+        """ Announce something in a channel """
+        if embed:
+            view = ConfirmView()
+            await ctx.respond("Are you sure? Embeds don't actually send pings to any roles or users", view=view, ephemeral=True)
+            await view.wait()
+            if not view.confirmed:
+                return
+        try:
+            if not embed:
+                await channel.send(message)
+            else:
+                embed = discord.Embed(colour=discord.Color.random(), description=message)
+                await channel.send(embed=embed)
+            if ctx.channel is not channel:
+                await ctx.respond("Message successfully sent!", ephemeral=True)
+        except Exception as e:
+            if str(e).startswith("403"):
+                e = "Missing Permissions"
+            await ctx.respond(f"Could not send message, `{e}`!\nMake sure I have view and write access in {channel.mention}", ephemeral=True)
```

Comment:
Please check if the bot has access to the channel before trying to send anything. Doing this based on errors isn't really good practise.